file-formats/sush/type/zif_aff_sush_v1.intf.abap

```diff
@@ -0,0 +1,186 @@
+  PUBLIC .
+
+  types:
+    "! <p class="shorttext">Maintenance Mode</p>
+    "! Maintenance Mode
+    "! $values {@link zif_aff_sush_v1.data:co_usob_status_flag}
+    ty_usob_status_flag type c length 1,
+    "! <p class="shorttext">Maintenance Status</p>
+    "! Maintenance Status
+    "! $values {@link zif_aff_sush_v1.data:co_okflag}
+    "! $default {@link zif_aff_sush_v1.data:co_okflag.y_flag}
+    ty_okflag           type c length 1.
+
+  constants:
+    "! <p class="shorttext">Maintenance Mode</p>
+    "! Maintenance Mode
+    begin of co_usob_status_flag,
+      "! <p class="shorttext">Manual Maintenance</p>
+      "! Manual Maintenance
+      manual type ty_usob_status_flag value space,
+      "! <p class="shorttext">Autom. Maintenance (All Objects)</p>
+      "! Autom. Maintenance (All Objects)
+      a_flag type ty_usob_status_flag value 'A',
+      "! <p class="shorttext">Automatic Maintenance (Basis Authorization Objects Only)</p>
+      "! Automatic Maintenance (Basis Authorization Objects Only)
+      b_flag type ty_usob_status_flag value 'B',
+      "! <p class="shorttext">Application Does Not Require Default Values</p>
+      "! Application Does Not Require Default Values
+      i_flag type ty_usob_status_flag value 'I',
+      "! <p class="shorttext">Application is Deprecated</p>
+      "! Application is Deprecated
+      d_flag type ty_usob_status_flag value 'D',
+      "! <p class="shorttext">Application Is Obsolete</p>
+      "! Application Is Obsolete
+      o_flag type ty_usob_status_flag value 'O',
+    end of co_usob_status_flag,
+
+    "! <p class="shorttext">Maintenance Status</p>
+    "! Maintenance Status
+    begin of co_okflag,
+      "! <p class="shorttext">No Default</p>
+      "! No Default
+      x_flag type ty_okflag value 'X',
+      "! <p class="shorttext">Default With Field Values</p>
+      "! Default With Field Values
+      y_flag type ty_okflag value 'Y',
+      "! <p class="shorttext">Default Without Field Values</p>
+      "! Default Without Field Values
+      v_flag type ty_okflag value 'V',
+      "! <p class="shorttext">Default Inactive</p>
+      "! Default Inactive
+      i_flag type ty_okflag value 'I',
+      "! <p class="shorttext">Unmaintained Object Status</p>
+      "! Unmaintained Object Status
+      u_flag type ty_okflag value 'U',
+      "! <p class="shorttext">Authorization Check Inactive</p>
+      "! Authorization Check Inactive
+      n_flag type ty_okflag value 'N',
+    end of co_okflag.
+
+  types:
+    "! <p class="shorttext">Application Name</p>
+    "! Name of Application
+    ty_name type c length 30,
+    "! <p class="shorttext">Application Type</p>
+    "! Type of Application
+    ty_type type c length 2.
+
+  types:
+    "! <p class="shorttext">General</p>
+    "! General
+    "! $required
+    begin of ty_general,
+      "! <p class="shorttext">Application Name</p>
+      "! Application Name of Authorization Default Value.
+      "! $required
+      name   type ty_name,
```

Comment:
Maybe, my comments are confusing. I hope not. ;)

I think here. I really would add something to make clear that the name is related to the application, not to the authorization values.